### PR TITLE
cloud: add windows ca path

### PR DESCRIPTION
**Pull request by Daemonxiao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add windows ca root in secure-connections-to-serverless-tier-clusters.md.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhangyangyu — APPROVED** [review]

**Review by shizn — APPROVED** [review]

File: tidb-cloud/secure-connections-to-serverless-tier-clusters.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ However, some drivers and ORMs do not use the system root CA stores. In those ca
 > However, TiDB Serverless Tier ensures always using a CA root certificate that is commonly available, which is provided in all common systems.
 >
 > If you really need the CA certificate of a TiDB Serverless Tier cluster, it is recommended that you download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of the single CA certificate in case we change the CA in the future.
+>
+> If you are using some GUI clients, like DBeaver, which does not accept a cert file with multiple certs inside. In such a case, you have to download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.
```
Comment (ran-huang) [inline tidb-cloud/secure-connections-to-serverless-tier-clusters.md:34]:
```suggestion
> If you are using a GUI client, such as DBeaver, which does not accept a certificate file with multiple certificates inside, you must download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.
```

File: tidb-cloud/secure-connections-to-serverless-tier-clusters.md [hunk 2]
```diff
@@ -160,6 +162,10 @@ The following lists the CA root paths on common platforms.
 /etc/ssl/ca-bundle.pem
 ```
 
+**Windows**
+
+Windows does not offer a specific path to CA root, instead, it uses [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.
```
Comment (ran-huang) [inline tidb-cloud/secure-connections-to-serverless-tier-clusters.md:167]:
```suggestion
Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.
```

File: tidb-cloud/secure-connections-to-serverless-tier-clusters.md [hunk 3]
```diff
@@ -160,6 +162,10 @@ The following lists the CA root paths on common platforms.
...
+
```
Comment (ran-huang) [inline tidb-cloud/secure-connections-to-serverless-tier-clusters.md:168]: Can we provide more detailed steps? For example, how can I point at this file for the drivers?

Comment (zhangyangyu) [reply tidb-cloud/secure-connections-to-serverless-tier-clusters.md:168]: It's the same as other OS. How to point it is drivers' work and we have already described it in previous section.

Comment (ran-huang) [reply tidb-cloud/secure-connections-to-serverless-tier-clusters.md:168]: Got it. Can you please link to that section? You know, people don't always read docs linearly and might skip some sections and thus get frustrated.

File: tidb-cloud/secure-connections-to-serverless-tier-clusters.md [hunk 4]
```diff
@@ -160,6 +162,10 @@ The following lists the CA root paths on common platforms.
 /etc/ssl/ca-bundle.pem
 ```
 
+**Windows**
+
+Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.
```
Comment (ran-huang) [inline tidb-cloud/secure-connections-to-serverless-tier-clusters.md:167]: How about this? It's easier to follow instructions. @zhangyangyu 

```suggestion
Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, to specify the CA root path on Windows, take the following steps:

1. Download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) and save it in a path you prefer, such as `<path_to_mozilla_ca_cert_bundle>`.
2. Use the path (`<path_to_mozilla_ca_cert_bundle>`) as your CA root path when you connect to a Serverless Tier cluster.
```

Comment (zhangyangyu) [reply tidb-cloud/secure-connections-to-serverless-tier-clusters.md:167]: LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**